internal/web/gist.go

```diff
@@ -186,8 +186,14 @@ func revisions(ctx echo.Context) error {
 		return errorRes(404, "Page not found", nil)
 	}
 
+	emailsUsers, err := models.GetUsersFromEmails(emails)
```

Comment:
Shouldn't emails be made into a set before using it to lookup? Feels wrong to send a list of 100 of the same emails imo, but maybe there's no performance cost at all.